keyboards/ergodox_ez/keymaps/ericgebhart/rules.mk

```diff
@@ -0,0 +1,17 @@
+SRC += ericgebhart.c
```

Comment:
It would be simpler/easier if this was in the rules.mk for the userspace.

Reply:
Ok. I just refactored all of the rules.mk files so I only have 2 now. One for
everyone, and one that adds rgblights to the xd75.